src/huggingface_hub/repocard_data.py

```diff
@@ -381,8 +378,7 @@ def __init__(
         paperswithcode_id: Optional[str] = None,
         pretty_name: Optional[str] = None,
         train_eval_index: Optional[Dict] = None,
-        configs: Optional[Union[str, List[str]]] = None,
-        ignore_metadata_errors: bool = False,
+        config_names: Optional[Union[str, List[str]]] = None,
```

Comment:
```suggestion
        config_names: Optional[Union[str, List[str]]] = None,
        ignore_metadata_errors: bool = False,
```

Reply:
I don't know if removing `ignore_metadata_errors` was on purpose but I'm adding it back.